Terraform/GCP/variables.tf

```diff
@@ -185,3 +185,9 @@ variable "k3s_port" {
   default = "6443"
 }
 
+variable "uptime_source_ips" {
```

Comment:
can you also add it as a placeholder to `gcp.tfvars.template` with an empty array, so we have the vars file "documented" :)